src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,55 @@
+footer {
+    text-align: center;
```

Comment:
you may want to find a way to update your linter / prettier to organize these in aphabetical order, but also that's a pretty big nitpick so you can choose to ignore